src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -85,6 +86,38 @@ internal bool TryUnparse(StringBuilder sb)
             }
         }
 
+        /// <summary>
+        /// Defines the different type of stop words remover supported.
+        /// </summary>
+        public enum StopWordsRemoverType
```

Comment:
>StopWordsRemoverType [](start = 20, length = 20)

I think it would be much cleaner if we just introduce IStopWordRemover interface and inherit both stop word removers from it and make it part of public interface.

Reply:
Interface make it difficult to find out different type of `StopWordsRemover` in the system at API level. When user will use this enum they immediately know that what type of `StopWordsRemover` are available and how to use them.

---
In reply to: [265808886](https://github.com/dotnet/machinelearning/pull/2962#discussion_r265808886) [](ancestors = 265808886)

Reply:
I have done what you proposed for the interface thing.

---
In reply to: [265811781](https://github.com/dotnet/machinelearning/pull/2962#discussion_r265811781) [](ancestors = 265811781,265808886)